pkg/cmd/taskrun/delete_test.go

```diff
@@ -18,7 +18,9 @@ import (
 	"io"
```

Comment:
These edits to `delete_test.go` aren't really testing anything. We have sufficient testing to see if TaskRun sorting is working, so it's all right to not include explicit unit tests in this case. This would probably be a better candidate for an `e2e` test, but it's all right to hold off on that for now as this feature will likely be changed to address other issues.

Reply:
Done